Server/src/main/java/com/guphub/CodeReviewAssignmentSubmissionApp/Dto/CommentResponseDTO.java

```diff
@@ -0,0 +1,55 @@
+package com.guphub.CodeReviewAssignmentSubmissionApp.Dto;
+
+import com.guphub.CodeReviewAssignmentSubmissionApp.Datamodels.Comment;
+
+public class CommentResponseDTO {
+
+    private Long id;
+    private String commentText;
+    private String username;
+
+
+
+    public CommentResponseDTO(Comment comment) {
+        this.id = comment.getId();
+        this.commentText = comment.getCommentText();
+        this.username = comment.getCreatedBy().getUsername();
+    }
+
+    public CommentResponseDTO() {
```

Comment:
Should we remove this if is empty?

Reply:
@CDDR1 when I get home I will go through my code again and let you know but I think the code is safe to merge otherwise so if anything I will push the fix to master